free-programming-books.md

```diff
@@ -1293,8 +1290,7 @@
 * [The ABC of PDF with iText: PDF Syntax essentials](https://leanpub.com/itext_pdfabc)
 * [The Java EE6 Tutorial](http://docs.oracle.com/javaee/6/tutorial/doc/javaeetutorial6.pdf) (PDF)
 * [The Java EE7 Tutorial](https://docs.oracle.com/javaee/7/JEETT.pdf) (PDF)
-* [The Java Language Specification](http://docs.oracle.com/javase/specs/) - James Gosling, Bill Joy, Guy Steele, Gilad Bracha
-* [The Java Tutorial 4th Edition](http://download.oracle.com/javase/tutorial/) - Sharon Zakhour, Scott Hommel, Jacob Royal, Isaac Rabinovitch, Tom Risser, Mark Hoeber
+* [Java Language and Virtual Machine Specifications](http://docs.oracle.com/javase/specs/) - James Gosling, Bill Joy, Guy Steele, Gilad Bracha
```

Comment:
I see the reasons for this change, but now there are two books at http://docs.oracle.com/javase/specs/ with different title and author lists (some omitted here). I tried adding a sublist - see https://github.com/vhf/free-programming-books/tree/java_manuals but it doesn't pass linter. Another solution would be to create two listings (with same url). (I looked at this first because it need an alphabetization tweak.) Any ideas? What does @vhf think?